src/Val.h

```diff
@@ -906,6 +906,10 @@ class TableVal final : public Val, public notifier::detail::Modifiable {
 	[[deprecated("Remove in v4.1.  Use Find() or FindOrDefault().")]]
 	Val* Lookup(Val* index, bool use_default_val = true);
 
+	// Returns true if this is a table[subnet]/set[subnet] and the
+	// given address was found in the table. Otherwise returns false.
+	bool Contains(const IPAddr& addr);
```

Comment:
since this is a new function - perhaps use the doxygen-style for usage. I think we use it everywhere else.

Reply:
Also - might it make sense to make it throw an error when this is called on anything but a table/set[subnet]? Just like LookupSubnets/LookupSubnetValues?